core/bessd_test.cc

```diff
@@ -116,11 +117,12 @@ TEST(TryAcquirePidfileLock, AlreadyHeld) {
     ASSERT_EQ(0, flock(fd, LOCK_EX | LOCK_NB)) << "Couldn't acquire file lock for test.";
 
     // Sleep for up to 10 seconds, while holding lock.
-    sleep(10);
+    sleep(3);
```

Comment:
Would it be possible to remove this arbitrary (and potentially unreliable) delay, e.g., with some IPC? A few second delay makes me a little hesitant to run the ./all_test script frequently.